src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 unreleased
 +++++++++++++++++++
+* `vm/vmss create`: don't throw when failed to retrieve the image to extract out the plan information 
```

Comment:
rec: "Fix issue where the command would throw an error if unable to extract plan information from an image."